src/index.html

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Nodes Engineering Blog
+---
+
+<div class="container">
```

Comment:
Let's get this indented correctly (4 spaces as the other files)

Reply:
Spaces or tabs?

Reply:
I just talked with Themi and since the frontend team is doing 2 spaces, we should do the same here.

Reply:
Now there are two spaces between Jekyll content and and html but I looked at posts and it doesn't seem that the indentation has been followed in old posts.